featurebyte/models/use_case.py

```diff
@@ -20,19 +20,31 @@ class UseCaseModel(FeatureByteCatalogBaseDocumentModel):
 
     context_id: PydanticObjectId
         The context id of ContextModel
-    target_id: PydanticObjectId
+    target_id: Optional[PydanticObjectId]
         The target id of TargetModel
+    target_namespace_id: Optional[PydanticObjectId]
```

Comment:
i think this doesn't have to be optional since we'll always have a target namespace ID.

Reply:
good idea. I will make it non-optional

Reply:
can update docstring to be non-optional too